src/Okta.Idx.Sdk/IdxClientExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="IdxClientExtensions.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Idx.Sdk
+{
+    internal static class IdxClientExtensions
+    {
+        internal static async Task<IIdxResponse> ContinueWithRemediationOption(this IIdxResponse response, string remediationType, IdxRequestPayload request, CancellationToken cancellationToken = default)
```

Comment:
Let's use the `Async` suffix to indicate this is an async method. Also, I wonder if `ProceedWithRemediationOptionAsync` makes more sense to indicate that internally we're calling the Proceed method, thoughts?